src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -141,25 +142,39 @@ private Contact buildPersonContact(Map<String, Object> pairs, String personType)
     }
 
     private String findHearingType(Map<String, Object> pairs) {
-        if (areBooleansValid(pairs, errors, "is_hearing_type_oral", "is_hearing_type_paper") && !doValuesContradict(pairs, errors, "is_hearing_type_oral", "is_hearing_type_paper")) {
+        if (areBooleansValid(pairs, errors, true, "is_hearing_type_oral", "is_hearing_type_paper") && !doValuesContradict(pairs, errors, "is_hearing_type_oral", "is_hearing_type_paper")) {
             return Boolean.parseBoolean(pairs.get("is_hearing_type_oral").toString()) ? "oral" : "paper";
         }
         return null;
     }
 
     private HearingOptions buildHearingOptions(Map<String, Object> pairs) {
 
-        String isLanguageInterpreterRequired = convertBooleanToYesNoString(findBooleanExists(getField(pairs,"hearing_options_language_type")));
+        boolean isSignLanguageInterpreterRequired = areBooleansValid(pairs, errors, false,"hearing_options_sign_language_interpreter") ? (boolean) pairs.get("hearing_options_sign_language_interpreter") : false;
```

Comment:
It would easier to read if we could use if else here rather than ternary operator ?